book/website/reproducible-research/open/open-hardware.md

```diff
@@ -1,111 +1,359 @@
+Your project may not need all of them, but it is worth considering having at least minimal information for most of these elements:
+
+- A **context description** which may reflects project maturity, complexity, the intentions of authors on how it should be used, and technical specifications. It may include standards compliance (the [DIN-SPEC](https://gitlab.com/OSEGermany/OHS-3105) standard for instance) and an estimation of the overall required budget and build time.
+
+- A **Bill of Materials** (BOM): a list or spreadsheet describing part numbers, putative suppliers, costs, and a short description. Make it easy to tell which item in the bill of materials corresponds to which component in your design files: use matching reference designators in both places, provide a diagram indicating which part goes where, or otherwise explain the correspondence.
+
+- **Assembly instructions**. To help others make and modify your hardware design, you should provide instructions for going from your design files to the working physical hardware. It is good to publish annotated photographs (or video) from multiple viewpoints and at various stages of assembly. If you do not have photos, posting annotated 3D renderings of your design is a good alternative. Either way, it is good to provide captions or text that explain what is shown in each image and why it is useful.
+
+    - **Manufacturing instructions**: The manufacturing process of parts that have been made for this project should be documented as well. This is specially important if they are available for purchase from only a handful of small/medium businesses.
+
+- A **list of required tools** and associated settings, for both software used for development, and machine tools for replication.
+
+- **Design files** with parts metadata, typically including the manufacturing process, the materials with dimensions, mass, and units. A functional overview of the project's parts/modules can also be included. Ideally, your OSH project would be designed using a free and open source software application, to maximize the ability of others to view and edit it. It is essential to share these original design files, in both original and accessible ready-to-view formats. The type of parts can also be mentionned with unambiguous reference, between, custom parts (e.g. developed as a result of this or another project), off-the-shelf parts (e.g. screws) or (maybe proprietary) complex modules (for example, a single board computer). 
+
+-  **Software and firmware**. You should share any code or firmware required to operate your hardware. This will allow others to use it with their hardware or modify it along with their modifications to your hardware. Document the process required to build your software, including links to any dependencies (for example, third-party libraries or tools). In addition, it is helpful to provide an overview of the state of the software (for example, “stable” or “beta” or “barely-working hack”). 
```

Comment:
```suggestion
-  **Software and firmware**. You should share any code or firmware required to operate your hardware. 
This will allow others to use it with their hardware or modify it along with their modifications to your hardware.
Document the process required to build your software, including links to any dependencies (for example, third-party libraries or tools). 
In addition, it is helpful to provide an overview of the state of the software (for example, “stable” or “beta” or “barely-working hack”). 
```